package.json

```diff
@@ -22,18 +20,13 @@
     "eject": "react-scripts-ts eject"
   },
   "devDependencies": {
-    "@types/chai": "4.1.4",
-    "@types/enzyme": "3.1.13",
-    "@types/enzyme-adapter-react-15": "1.0.3",
-    "@types/enzyme-adapter-react-16": "latest",
+    "@angular/core": "^8.2.14",
```

Comment:
Angular?

Reply:
This is a temporary requirement of `@mindspace-io/utils`. I will fix later today.